src/tui/mode/helix.zig

```diff
@@ -152,6 +154,28 @@ const cmds_ = struct {
     }
     pub const @"bc!_meta": Meta = .{ .description = "bc! (Close buffer/tab forcefully, ignoring changes)" };
 
+    pub fn @"bco!"(_: *void, _: Ctx) Result {
+        const mv = tui.mainview() orelse return;
+        if (tui.get_buffer_manager()) |bm| {
+            if (mv.get_active_buffer()) |buffer| bm.delete_others(buffer);
+        }
+    }
+    pub const @"bco!_meta": Meta = .{ .description = "bco! (Close other buffers/tabs forcefully, ignoring changes)" };
+
+    pub fn bco(_: *void, _: Ctx) Result {
+        const logger = log.logger("helix-mode");
+        defer logger.deinit();
+        const mv = tui.mainview() orelse return;
+        const bm = tui.get_buffer_manager() orelse return;
+        if (mv.get_active_buffer()) |buffer| {
+            const remaining = bm.close_others(buffer);
+            if (remaining > 0) {
+                logger.print("{} unsaved buffer(s) remaining", .{remaining});
+            }
+        }
+    }
+    pub const bco_meta: Meta = .{ .description = "bco (Close other buffers/tabs, except this one)" };
```

Comment:
This can be "bco (Close other buffers/tabs)".  The exception is implied already by "other".